Pipfile

```diff
@@ -1,17 +1,31 @@
 [[source]]
-name = "pypi"
 url = "https://pypi.org/simple"
 verify_ssl = true
-
-[dev-packages]
-pytest = "~=6.2.4"
+name = "pypi"
 
 [packages]
```

Comment:
Interesting. I really don't understand where or how the template configurator in invoked. But I don't have it in my global environment and haven't seen it as a requirement in any other LinkML repo I use. Let's meet so you can show me how it's used.

Reply:
The ` template-configurator` is invoked in the [MakeConfig](https://github.com/cancerDHC/ccdhmodel/blob/issue-69-pypi/from_template/MakeConfig) file, specifically on line [7](https://github.com/cancerDHC/ccdhmodel/blob/issue-69-pypi/from_template/MakeConfig#L7) and line [10](https://github.com/cancerDHC/ccdhmodel/blob/issue-69-pypi/from_template/MakeConfig#L10). The `configure` CLI is actually part of that package. But yes, I'll find some time for us to chat about this.

Hm, I don't think it's a dependency from any other packages that you're including either, since I don't see the package in the `Pipfile.lock` file.